bower.json

```diff
@@ -13,7 +13,15 @@
     "package.json"
   ],
   "dependencies": {
-    "d2l-colors": "^2.2.3",
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#^1.9.1 || ^2.0.0",
```

Comment:
Yes - i think the || is required for hybrid components - the variants are only for development purposes.  For regular use it will only use the regular bower dependencies, so you have to ensure that those dependencies all have hybrid support too. And then make the Polymer version conditional so that the consumer can use either 1.9 or 2.